errors.go

```diff
@@ -0,0 +1,9 @@
+package sriovnet
```

Comment:
copyright

Reply:
There is a project level license file.

do we need for every file as well ?
most files in this proj dont have a license.

Reply:
so maybe it better to disable the copyright check in that case